src/logging/utils.ts

```diff
@@ -34,3 +36,23 @@ export function formatRouteLogMessage(
     detail: detail ?? null,
   };
 }
+
+export const asError = (thrown: unknown): Error => {
```

Comment:
This is a helper [copied over from the web](https://github.com/safe-global/safe-wallet-web/blob/946d56e1273317cdde33a324109aefd070b14827/src/services/exceptions/utils.ts#L1) for extracting the error `message` when in strict mode.

Reply:
Added in fc58887c6782d4416d3ac7505c59a57ffa7e817c.